Algorithm.CSharp/DropboxBaseDataUniverseSelectionAlgorithm.cs

```diff
@@ -30,8 +30,10 @@ namespace QuantConnect.Algorithm.CSharp
     /// <meta name="tag" content="using data" />
     /// <meta name="tag" content="universes" />
     /// <meta name="tag" content="custom universes" />
-    public class DropboxBaseDataUniverseSelectionAlgorithm : QCAlgorithm, IRegressionAlgorithmDefinition
+    public class DropboxBaseDataUniverseSelectionAlgorithm : QCAlgorithm//, IRegressionAlgorithmDefinition
     {
+        // Regression algorithm disabled due to dropbox file missing
```

Comment:
I got a response -- they said my account has been deleted along with all of the files :(